<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashTransactionDialog</class>
 <widget class="QDialog" name="CashTransactionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>142</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="radioButtonHorizontalLayout">
       <item>
        <widget class="QRadioButton" name="incomeRadioButton">
         <property name="text">
          <string>Income</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="expenseRadioButton">
         <property name="text">
          <string>Expense</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="amountLabel">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="amountDoubleSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <double>10000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSelect_Payee">
   <property name="text">
    <string>Select Payee</string>
   </property>
  </action>
  <action name="actionSelect_Tag">
   <property name="text">
    <string>Select Tag</string>
   </property>
  </action>
  <action name="actionSelect_Account">
   <property name="text">
    <string>Select Account</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>incomeRadioButton</tabstop>
  <tabstop>expenseRadioButton</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>amountDoubleSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
